src/lib/CLValue2/Key.ts

```diff
@@ -18,7 +25,7 @@ export class CLKeyType extends CLType {
 // TBD: Maybe the first should be CLByteArray insted?
```

Comment:
I like the implementation, but just want to make a note, that Key type has grown in the last 2 months. https://docs.rs/casper-types/1.0.1/casper_types/enum.Key.html Let's keep it as is, but revisit soon in future, because Key might change even more.